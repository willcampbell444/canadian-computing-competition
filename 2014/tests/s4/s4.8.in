100
5000000
0 0 1000000000 1000000000 778131
136033743 396501386 150016439 973467274 34466
396447864 604960447 976071020 991032286 142763
548998595 82310789 607323086 429621846 940634
337203221 165174108 576696213 939536174 344426
74228530 456878766 488856151 725875014 877069
14440064 821482116 477540243 872458456 911470
38262695 130423817 896583311 776482990 468690
383451490 70750977 553196841 915577402 559591
22444440 59559787 293943801 101058500 623302
516622547 391741186 602229979 588811936 546170
207310522 72888120 429905093 842446398 444367
83585783 164928321 341881349 481060111 333201
231954603 672526189 637176534 778176510 202941
100755496 457486192 494509384 973486014 955878
439205072 286229493 836756331 611328943 702765
571621219 126098947 599195391 543457895 869153
289364916 98661336 343681862 677511044 798693
360738285 374999092 673222433 905562228 213878
19542977 479790411 559509576 994020459 321771
242119360 467052928 481477090 611551034 814305
68824946 380955551 253664724 833155653 172529
388497643 108178716 508752013 922577599 753632
889956700 278638739 945747363 294953426 919107
24397232 460875610 617451382 864378164 841120
55915028 24040985 537406604 106500434 883223
531219534 117170552 879509037 778425752 941744
119813830 518846764 834855928 912899238 450660
460512316 24572166 857499515 92409802 844012
216883356 716217517 721786063 878798527 949978
427961603 16286065 898445935 827911012 812965
357153539 329823517 893409621 462719802 540506
268125976 160371281 658377499 972570420 994817
102803851 685961886 564940964 730557139 184100
106747008 276890809 222086243 948101405 636289
45972235 708707565 326265429 810292990 717272
227302121 40806473 468010074 405693336 598635
138873666 527958752 327604683 696814852 298654
710018708 332950850 724493678 510777556 229827
113739965 33857883 756371992 953119888 413222
98673823 138567152 630313287 553550871 833510
185067647 503251845 185556585 668600851 140223
377222324 396852102 942164959 569519775 49311
846508316 268130792 847292153 979084216 698192
100906452 570124474 802980183 900681127 456980
292780234 619123616 560194410 872340755 327591
496093501 104932230 941972085 268365489 889387
840394056 295931451 940749084 395228767 945377
72100647 53287538 102953561 616147085 890340
595468699 561607651 878888601 881171519 140371
308846274 311335513 773062948 592885880 786093
329078040 366966240 398558665 837205833 540075
451657155 440090519 667655759 688269132 361976
298412467 44000317 571931829 114424609 517977
11308250 517687131 258069286 600989702 339914
8397850 201722827 65248631 417064440 341617
625368108 5368817 751681097 569801316 114618
453855668 340090906 999031460 883276511 335507
796120691 222827759 953945005 988257373 131986
294731704 103954078 911007053 884603501 967129
279238170 814778930 766746991 904137689 836045
409599975 107186122 885839461 592674955 797093
153226202 232862839 803508817 988493561 765114
175190541 227325744 612477221 808511854 354589
35532541 194945915 906714533 607180213 414924
373742470 100009009 719638749 340537733 185503
517652931 529535898 530583096 950970706 849700
472497311 660757950 857600758 933167882 910207
27179602 33826642 235095869 186532582 447895
260652969 75276942 897670074 861292121 391799
470258673 690674729 814036846 691934264 209737
15686628 571738682 64060693 652135055 313676
822114765 166049135 999533070 180715588 993852
281991747 93004343 757014377 731861479 364626
562838945 723241941 932846664 761662159 869740
236243657 630049919 532144462 642858475 103855
840005033 407077447 950360311 515359170 219228
667783241 595926610 824951352 683375459 778451
72209142 262569384 465237731 876763002 967479
326436161 739306682 980001411 831609027 320692
332080124 108112354 859142279 717219304 114911
16475004 133467985 568099767 455975728 902932
555938051 136554322 828058850 742453639 304797
33811832 339760809 733080140 764213335 878069
704494585 8639116 980882511 607871198 201457
168823246 586381029 503481514 928219812 151670
651649352 769803841 824518879 897037943 489255
656091816 399602746 941971047 457924439 122266
527338613 108036100 932029563 668122701 224123
885813946 40010842 946920560 163844350 938190
573844121 249066164 609373539 391540624 356354
111574738 108125039 842542709 276376313 348305
24922389 376540237 733184847 697254721 812747
491099071 661953734 966851772 915409454 824849
78054911 342354823 641798426 688765480 548224
410957782 12966863 734453263 453011900 196673
334321147 409270212 749541760 911275655 314937
574425947 390461800 661053334 437881157 126009
9699665 593225773 747951153 960979323 972369
83680763 153404598 414692560 406768919 751811
