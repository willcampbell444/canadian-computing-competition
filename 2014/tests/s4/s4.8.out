648073624209112575
